### ISSUE TITLE
DISABLE_AUTH no longer working

### ISSUE THREAD

**Issue by monken** [rocker-org/rocker-versioned2#78]
with the latest rstudio release it seems that DISABLE_AUTH is no longer working. To reproduce, clear all cookies for localhost and launch the container with DISABLE_AUTH=true. The server will launch but the website will be stuck launching an R session. Funny enough, if you are coming from a session WITH authentication, it will work because a cookie was set.

<img width="435" alt="Screen Shot 2020-08-12 at 6 23 29 PM" src="https://user-images.githubusercontent.com/46471/90074323-ee6e5700-dcc8-11ea-8955-aa6cc71ef31d.png">

**Comment by vpanfilov** [comment]
Related to https://github.com/rstudio/rstudio/issues/7575 (`--auth-none=1` no longer working).

@monken thanks for reporting this issue to RStudio team.

**Comment by cboettig** [comment]
Try upgrading your instance to RStudio daily. e.g. launch as a bash session, upgrade and then launch:

```
docker run --rm -ti -e PASSWORD=sometest -p 8787:8787 rocker/rstudio bash
RSTUDIO_VERSION="daily" /rocker_scripts/install_rstudio.sh && /init
```

or just `docker-exec` into a running instance and run:

```
RSTUDIO_VERSION="daily" /rocker_scripts/install_rstudio.sh && /init
```

   * NOTE: requires an up-to-date rocker image pull.

**Comment by Pit-Storm** [comment]
Is this still the case? It looks like that this is fixed or outdated. Can someone still reproduce it with actual rstudio versions?

**Comment by vpanfilov** [comment]
@Pit-Storm It's a bug in Rstudio which was fixed in RStudio 1.4. I had the same issue with RStudio 1.3 versions greater than 1.3.959.
